Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -4216,6 +4210,10 @@ static void handleAnimationActions(Element& element, uint32_t action)
             m_viewportConfiguration.setDefaultConfiguration(ViewportConfiguration::imageDocumentParameters());
         else if (document->isTextDocument())
             m_viewportConfiguration.setDefaultConfiguration(ViewportConfiguration::textDocumentParameters());
+#if ENABLE(PDF_PLUGIN)
+        else if (document->isPluginDocument())
```

Comment:
We want to guard this behavior behind the feature flag, right?
```suggestion
        else if (m_page->settings().unifiedPDFEnabled() && document->isPluginDocument())
```

Reply:
`isPluginDocument()` should always be false without the FF enabled, but still.

Reply:
Oh nice! I think that still leads to build errors on certain platforms since `pluginDocumentParameters` is behind `ENABLE(PDF_PLUGIN)`? :/

Reply:
Oh oops I think I misread the suggestion! This is *in addition* to the compile time guard